packaging/bundle-validation/ci_run.sh

```diff
@@ -27,10 +27,29 @@
 # This is to run by GitHub Actions CI tasks from the project root directory
 # and it contains the CI environment-specific variables.
 
-HUDI_VERSION=$1
+HUDI_VERSION=0.12.2-rc1
+REPO_BASE_URL=https://repository.apache.org/content/repositories/orgapachehudi-1106/org/apache/hudi
```

Comment:
we set these 2 vars to download hudi bundle jars directly from apache's artifactory. `REPO_BASE_URL` should be changed accordingly wrt `HUDI_VERSION`, which can be any release candidate. These info is available in the RC voting email